lib/safe_active_record/load.rb

```diff
@@ -70,6 +116,7 @@ def self.apply_load_patch(safe_query_mgr)
           # decorated.
 
           abspath = "#{abspath}.rb" unless File.exist?(abspath)
+          abspath = "#{abspath[...-3]}.so" unless File.exist?(abspath)
```

Comment:
What's the purpose of this line? Did it not work as intended before?

Reply:
Yes, there was a bug before. Ruby can import either ".rb" files, which is the most common, but also ".so" file, e.g. in the case of a C extension. A unit test for .so files is also part of this PR.

Reply:
Ack. Add a comment so there's context for others.

Reply:
Done :)